satellite/peer.go

```diff
@@ -360,21 +417,38 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 			return nil, errs.Combine(err, peer.Close())
 		}
 
-		peer.Console.Service, err = console.NewService(peer.Log.Named("console:service"),
-			// TODO: use satellite key
+		mailService := peer.Mail.Service
+		// fallback to use account activation mail sender
+		// when using storj-sim
+		if config.SimulateActivation {
```

Comment:
The setup should be above. It's not really configuring console, but rather mail service.

Reply:
For me it's more like configuring console service to automatically activate account through mail service, but still console specific. This way you won't have to modify stimulative `MailSender` when there is any change in any of the services which uses `mailservice.Service`. If there were more services using `mailservice.Service`, then for storj-sim to work offline I would make global `mail.Config` option, for example to write all emails to file without connecting to any server. 
If I make console sim activation global, then `mailservice` will know about console implementation detail, which it shouldn't  in my opinion.

Reply:
Yeah, my mental model was a mail-sender that automatically clicks links.